packages/components/src/illustrated-message/docs/IllustratedMessage.stories.mdx

```diff
@@ -0,0 +1,79 @@
+import { ArgsTable, Meta, Story } from "@storybook/addon-docs";
+import { ComponentInfo, Preview, Tagline } from "@stories/components";
+import { Content } from "@components/placeholders";
+import { Heading } from "@components/typography";
+import { IllustratedMessage, InnerIllustratedMessage } from "@components/illustrated-message";
+import { Image, SvgImage } from "@components/image";
+import { ReactComponent as NoResults } from "./assets/no-results.svg";
```

Comment:
Should be imported directly from "./assets" instead of "./assets/no-results.svg"